src/LibBsonAPI.pas

```diff
@@ -7,11 +7,11 @@ interface
 
 const
   (* PLEASE!!! maintain this constant in sync with the dll driver version this code operates with *)
-  LibBson_DllVersion = '1-0-1';
```

Comment:
I've changed c driver to use dash: https://github.com/Convey-Compliance/mongo-c-driver-build/pull/17